src/wallet/fees.cpp

```diff
@@ -87,7 +87,7 @@ CFeeRate GetDiscardRate(const CWallet& wallet)
     CFeeRate discard_rate = wallet.chain().estimateSmartFee(highest_target, false /* conservative */);
     // Don't let discard_rate be greater than longest possible fee estimate if we get a valid fee estimate
     discard_rate = (discard_rate == CFeeRate(0)) ? wallet.m_discard_rate : std::min(discard_rate, wallet.m_discard_rate);
-    // Discard rate must be at least dustRelayFee
+    // Discard rate must be at least dust relay feerate
```

Comment:
`m_dust_relay_feerate` we do so for other variables doc refs make the code more greppable

Reply:
`wallet.m_dust_relay_feerate` doesn't exist, only `wallet.chain().relayDustFee()`. Happy to switch to `relayDustFee` or similar, or just remove the comment completely, as the code is self-explanatory. Just let me know.